src/spatial/views/PMcamsDataViews.py

```
@@ -0,0 +1,32 @@
+from models.PMcamsDataModel import CamsDataController
+from flask import request, jsonify
```

Ruff (0.11.9)

`flask.jsonify` imported but unused

Remove unused import: `flask.jsonify`

(F401)